### PR TITLE
Fix/sprint2 andrew

### PR DESCRIPTION
**Pull request by Merteg**

### CODE REVIEW

File: server/server/views/event.py [hunk 1]
```diff
@@ -48,18 +50,23 @@ def collection_post(self):
...
+            new_event.category_id = Category\
+                .get_by_name(request, request.validated['category'].lower()).id
+
             Event.add_event(request, obj=new_event)
             new_event_id = Event.get_event_obj(request,
                                                name=request.validated['name']
                                                ).id
+
+            if 'tags' in request.validated:
+                for tag in request.validated['tags']:
```
Comment (serge30) [inline server/server/views/event.py]: `dict` has very useful method [get](https://docs.python.org/2/library/stdtypes.html#dict.get). So you may substitute 2 lines above by one:
```
for tag in request.validated.get('tags', []):
```

**Comment by Merteg** [comment]
Update

**Review by yazhyvyi — COMMENTED** [review]
"Add Event": you need to make "main_image" a null value by default, it gives error on button click -- "Add Event"

**Comment by Merteg** [comment]
Sorry for the delay, update

**Review by ykobuk — APPROVED** [review]

**Review by yazhyvyi — APPROVED** [review]
Did some testing, no issues here. Any additions or changes should be applied to the next PR

**Pull request merged**